packages/x-components/src/x-modules/next-queries/components/next-queries-list.vue

```diff
@@ -104,30 +59,24 @@
     setup(props, { slots }) {
       const $x = use$x();
 
-      /**
-       * The state next queries.
-       *
-       * @internal
-       */
+      /** The state next queries. */
       const nextQueries: ComputedRef<NextQuery[]> = useGetter('nextQueries', [
         'nextQueries'
       ]).nextQueries;
 
-      /**
-       * Injected query, updated when the related request(s) have succeeded.
-       */
-      const injectedQuery = inject<Ref<string | undefined>>(QUERY_KEY as string);
+      /** Injected query, updated when the related request(s) have succeeded. */
+      const injectedQuery = inject<Ref<string> | undefined>(QUERY_KEY as string, undefined);
```

Comment:
```suggestion
      const injectedQuery = inject<Ref<string> | undefined>(QUERY_KEY as string);
```
Is there a reason to explicitly set `undefined` as the default? Vue already does that:

> If no value with matching key was found, inject() returns undefined unless a default value is provided.

https://vuejs.org/api/composition-api-dependency-injection.html#inject

Reply:
It was because at some point in the time, I saw a warning saying "you are injecting a key that wasn't provided". But I am not able to see it again. 
As you say, Vue does it by us, I restored the injected default value for the time being and also removed the `undefined` from the generic type because when you don't give a default value for the injection, `undefined` is already inferred `export declare function inject<T>(key: InjectionKey<T> | string): T | undefined;`